apps/metastore/src/metastore/tests.py

```diff
@@ -103,6 +103,41 @@ def test_show_tables(self):
     assert_equal(data['tables'], [{'name': 'customer'}, {'name': 'opportunities'}])
 
 
+  def test_show_tables_hs2(self):
```

Comment:
What is new test testing? (better name,           HIVE_DISCOVERY_HS2.return_value = True or False?, ...)